_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -13,64 +13,72 @@ Security Analytics provides the options and functionality to monitor and respond
+Although these automatic matches are normally dependable, it's still a good idea to review the mappings in the **Automatically mapped fields** table and verify that they are correct and matched as expected. If you find a mapping that doesn't appear to be accurate, you can use the dropdown list to search for and choose the correct field name. For more on matching field names, see the [Pending field mappings](#pending-field-mappings) section that follows.
 
 ### Pending field mappings
 
-The field names that are not automatically mapped appear in the **Pending field mappings** table. In this table you can manually map rule fields to log fields, as shown in the following image.
-<br><img src="{{site.url}}{{site.baseurl}}/images/Security/pending-mappings.png" alt="Field mapping example for pending mappings" width="85%">
+The field names that are not automatically mapped appear in the **Pending field mappings** table. In this table you can manually map rule fields to log source fields, as shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/pending-mappings.png" alt="Field mapping example for pending mappings" width="85%">
 
 While mapping fields, consider the following:
-* The Rule field name column lists field names based on all of the prepackaged rules associated with the selected log type.
-* The log field name column includes a dropdown list for each of the rule fields. Each dropdown list contains field names extracted from the log index.
-* To map a rule field name to a log field name, use the dropdown arrow to open the list of log fields and select the log field name from the list. To search for names in the log field list, enter text in the **Select a mapping field** box.
-<br><img src="{{site.url}}{{site.baseurl}}/images/Security/log-field.png" alt="Field mapping example for pending mappings" width="60%">
-* Once the log field name is selected and mapped to the rule field name, the icon in the Status column to the right changes to a green check mark.
-* Make as many matches between field names as possible to complete an accurate mapping for rule and log fields.
+* The **Detector field name** column lists field names based on all of the prepackaged rules associated with the selected log type.
+* The **Log source field name** column includes a dropdown list for each of the detector fields. Each dropdown list contains field names extracted from the log index.
+* To map a detector field name to a log source field name, use the dropdown arrow to open the list of log source fields and choose the log field name from the list. To search for names in the log field list, enter text in the **Select a mapping field** box.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/log-field.png" alt="Field mapping example for pending mappings" width="60%">
```

Comment:
Reference images in the text that precedes them. For example, "..., as shown in the following image."